src/PackagePrivate/Humanizer/InternationalizedHumanizer.php

```diff
@@ -91,11 +91,26 @@ private function humanizeSeason( Season $season ): string {
 		);
 	}
 
+	private function approximateMessage( array $info ) : string {
+		switch( count( $info ) ) {
+			case 1: return $this->message( 'edtf-approximate-one', $info[0] );
```

Comment:
You could do

`return match () {}`

https://www.php.net/manual/en/control-structures.match.php

Reply:
Actually NVM. This lib still supports PHP 7.4 :(